themes/qgrid/layout/index.ejs

```diff
@@ -1,30 +1,43 @@
 <div class="get-started">
-    <h1>qgrid</h1>
-    <h2>Fast and flexible data table solution. <br> Start with your ecosystem.</h2>
-    <a href="<%- url_for('guide/introduction.html') %>" class="angular-js--big"></a>
-    <a href="<%- url_for('guide/introduction.html') %>" class="angular--big"></a>
+    <div class="wrapper">
```

Comment:
Honestly i dont lile name "wrapper" it doesn't say anything about layout or semantics, can we just apply 60% in css?

Reply:
We have to implement additional div for centering of content. Why it should be separate? Well, one - for background. If we set 60% -- then bg won't be on the full width. If you don't like wrapper class, then we could use >div selector, but I don't like it because it says nothing about what it should be like

Reply:
ok what does mean "wrapper" from semantic point of view? You can make the div with valuable name, it's ok.